src/styles/mediaqueries.scss

```diff
@@ -12,16 +12,45 @@ For Mobile
   }
   .navBarText {
       font-size: 0.1rem;
+      padding-top: 0.5rem;
     }
     .navbarLogo {
       max-width: 15%;
       float: center;
+      padding-top: 1rem;
+    }
   .mission {
+    margin: 3rem 0;
+    padding: 0.5rem 0;
     .missionTitle {
       font-size: .8rem;
     }
     .missionDetails {
-      font-size: .7rem;
+      font-size: .65rem;
+    }
+  }
+  .broCol{
+    .bro1 {
+      width: 50%;
+    }
+    .stripes {
+      margin: -0.4rem 0 0 -1.75rem;
```

Comment:
So I think a better way to do this is do a position: absolute on the elements inside of broCol and position them to their exact position instead of messing around with the margins like this. Doing it with the margins leaves unintended consequences (look at what happens when you go into tablet or mobile mode. Using the margins like this leaves a bunch of space to the left and bottom that we don't want.)